packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -93,6 +103,75 @@ export class Copy {
         }
     }
 
+    /**
+     * Private function that checks if a dataset is type PDS
+    **/
+    public static async isPDS(
+        session: AbstractSession,
+        dataSetName: string
+    ): Promise<boolean> {
+        try {
+            const response = await List.dataSet(session, dataSetName, {attributes: true});
+            const dsorg = response.apiResponse.items[0].dsorg;
+            return dsorg === "POE" || dsorg === "PO";
+        }
+        catch(error) {
+            Logger.getAppLogger().error(error);
+            throw error;
+        }
+    }
+
+    /**
+     * Copy the members of a Partitioned dataset into another Partitioned dataset
+     *
+     * @param {AbstractSession}   session        - z/OSMF connection info
+     * @param {IDataSet}          toDataSet      - The data set to copy to
+     * @param {IDataSetOptions}   options        - Options
+     *
+     * @returns {Promise<IZosFilesResponse>} A response indicating the status of the copying
+     *
+     * @throws {ImperativeError} Data set name must be specified as a non-empty string
+     * @throws {Error} When the {@link ZosmfRestClient} throws an error
+     *
+     * @see https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.izua700/IZUHPINFO_API_PutDataSetMemberUtilities.htm
+     */
+
+    public static async copyPDS (
+        session: AbstractSession,
+        fromPds: string,
+        toPds: string
+    ): Promise<IZosFilesResponse> {
+        try {
+            const sourceResponse = await List.allMembers(session, fromPds);
+            const sourceMemberList: Array<{ member: string }> = sourceResponse.apiResponse.items;
+
+            if(sourceMemberList.length == 0) {
+                return {
+                    success: false,
```

Comment:
Should we add a unit test where the source PDS has no members?

And one more thing... would it be ok to return `success: true` here? 
Not a massive difference, but some user might feel like they did something wrong 😅 

The first one is with `success: false` and the second is `success: true`
![image](https://github.com/user-attachments/assets/0003fa1b-536f-4c38-986e-23e485e624cf)